Cargo.lock

```diff
@@ -7883,18 +7851,18 @@ dependencies = [
 
```

Comment:
We'll need to bump `spl-instruction-padding` as well. I'm not sure the motivation for using a less-specific version in `Cargo.toml` nor if it's important to keep that definition, but can run `cargo update spl-instruction-padding --precise 0.1.1` in the meantime.